clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/VCVTraitSet.scala

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.monster.etl.clinvar.models.output
+
+import io.circe.Encoder
+import io.circe.derivation.{deriveEncoder, renaming}
+import upack.Msg
+
+/**
+  * Info about a collection of traits in ClinVar.
+  *
+  * @param id unique ID of the trait collection
+  * @param `type` common type of the trait collection
+  */
+case class VCVTraitSet(id: String, `type`: Option[String])
```

Comment:
This needs a `traitIds` field (there are separate tickets for drawing the difficult lines between tables, but this one should be straightforward enough to do here)